<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editPerson</class>
 <widget class="QDialog" name="editPerson">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="l_name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_birthyear">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter year of birth:</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_gender">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose gender:</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_deathyear">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter year of death(if dead):</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_turing">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Turing year</string>
   </property>
  </widget>
  <widget class="QLabel" name="Contribution">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Contribution:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_name">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_birthYear">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>271</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbMale">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Male</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbFemale">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Female</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cB_TuringYear">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>79</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1801</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1802</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1803</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1804</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1805</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1806</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1807</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1808</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1809</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1810</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1811</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1812</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1813</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1814</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1815</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1816</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1817</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1818</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1819</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1820</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1821</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1822</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1823</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1824</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1825</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1826</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1827</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1828</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1829</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1830</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1831</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1832</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1833</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1834</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1835</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1836</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1837</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1838</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1839</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1840</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1841</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1842</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1843</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1844</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1845</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1846</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1847</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1848</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1849</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1850</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1851</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1852</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1853</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1854</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1855</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1856</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1857</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1858</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1859</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1860</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1861</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1862</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1863</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1864</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1865</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1866</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1867</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1868</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1869</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1870</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1871</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1872</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1873</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1874</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1875</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1876</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1877</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1878</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1879</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1880</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1881</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1882</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1883</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1884</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1885</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1886</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1887</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1888</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1889</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1890</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1891</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1892</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1893</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1894</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1895</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1896</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1897</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1898</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1899</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1900</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1901</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1902</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1903</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1904</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1905</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1906</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1907</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1908</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1909</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1910</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1911</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1912</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1913</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1914</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1915</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1916</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1917</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1918</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1919</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1920</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1921</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1922</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1923</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1924</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1925</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1926</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1927</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1928</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1929</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1930</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1931</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1932</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1933</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1934</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1935</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1936</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1937</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1938</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1939</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1940</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1941</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1942</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1943</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1944</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1945</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1946</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1947</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1948</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1949</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1950</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1951</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1952</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1953</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1954</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1955</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1956</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1957</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1958</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1959</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1960</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1961</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1962</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1963</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1964</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1965</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1966</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1967</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1968</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1969</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1970</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1971</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1972</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1973</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1974</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1975</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1976</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1977</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1979</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1980</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1981</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1982</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1983</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1984</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1985</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1986</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1987</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1988</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1989</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1990</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1991</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1992</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1993</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1994</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1995</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1996</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1997</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1998</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1999</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2001</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2002</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2003</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2004</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2005</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2006</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2007</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2008</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2009</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2010</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2011</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2012</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2013</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2014</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2015</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2016</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pBEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBCancel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>input_name</tabstop>
  <tabstop>input_birthYear</tabstop>
  <tabstop>rbMale</tabstop>
  <tabstop>rbFemale</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>cB_TuringYear</tabstop>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>pBEdit</tabstop>
  <tabstop>pBCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
